hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -381,11 +381,26 @@
     },
     { name: "rstmgr_aon",
       type: "rstmgr",
-      clock_srcs: {clk_i: "io_div4", clk_aon_i: "aon", clk_main_i: "main", clk_io_i: "io", clk_usb_i: "usb",
-                   clk_io_div2_i: "io_div2", clk_io_div4_i: "io_div4"},
+      clock_srcs: {clk_i: {
```

Comment:
sorry this is really nit-picky, but could you move the `clk_i` dict definition to the next line? similar to the `reset_connections` below.

Reply:
It's good idea. I updated it.